src/Rings/MPolyQuo.jl

```diff
@@ -365,6 +357,114 @@ function Oscar.addeq!(a::MPolyQuoElem, b::MPolyQuoElem)
   return a
 end
 
+@doc Markdown.doc"""
+    simplify(a::MPolyQuoIdeal)
+Simplify the ideal `a` w.r.t. the parent quotient ring. Replace inplace `a.SI`to the unique simplified representation.
```

Comment:
Implementation details like `a.SI` should not appear in the docstring.

If this mutates `a`, it should be `simplify!`.